### PR TITLE
Drops 512 support.

### PR DESCRIPTION
**Pull request by AnturK**
Alright, if you can think of any reasons against please respond since this was only briefly discussed.

@tgstation/commit-access If anyone missed this.

(ed. SpaceManiac) Closes #49360.

### CODE REVIEW

**Comment by SpaceManiac** [comment]
Is this pressing? Could we get away with waiting until 513 exits beta?

**Comment by optimumtact** [comment]
I was told by lummox that 512 isn't really getting any fixes. so it basically is out of beta, even if it's not official.

**Comment by SpaceManiac** [comment]
Alright, good enough for me

**Review by SpaceManiac — DISMISSED** [review]

**Comment by spookydonut** [comment]
fuk 512

**Review by spookydonut — DISMISSED** [review]

**Comment by Couls** [comment]
I was here

**Review by Couls — APPROVED** [review]

**Review by actioninja — DISMISSED** [review]
yeah, seems like it's time.

**Review by ShizCalev — APPROVED** [review]

File: code/__DEFINES/maths.dm [hunk 1]
```diff
@@ -30,11 +30,7 @@
 // round() acts like floor(x, 1) by default but can't handle other values
 #define FLOOR(x, y) ( round((x) / (y)) * (y) )
 
-#if DM_VERSION < 513
-#define CLAMP(CLVALUE,CLMIN,CLMAX) ( max( (CLMIN), min((CLVALUE), (CLMAX)) ) )
-#else
 #define CLAMP(CLVALUE,CLMIN,CLMAX) clamp(CLVALUE, CLMIN, CLMAX)
```
Comment (ShizCalev) [inline code/__DEFINES/maths.dm]: is this really needed anymore? just rename all instances of `CLAMP` to `clamp` imo

File: code/__DEFINES/maths.dm [hunk 2]
```diff
@@ -43,12 +39,7 @@
 #define MODULUS(x, y) ( (x) - (y) * round((x) / (y)) )
 
 // Tangent
-#if DM_VERSION < 513
-#define TAN(x) (sin(x) / cos(x))
-#else
 #define TAN(x) tan(x)
```
Comment (ShizCalev) [inline code/__DEFINES/maths.dm]: same

**Comment by SpaceManiac** [comment]
it's fine if the define -> real happens in a later PR IMO

**Comment by Dennok** [comment]
How does this affect performance?
Or it just removes the need to keep support 512

**Comment by SpaceManiac** [comment]
> it just removes the need to keep support 512

**Pull request merged**